modules/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@erento/nestjs-common",
-  "version": "7.1.0",
+  "version": "7.2.0-beta.0",
```

Comment:
do not change version in the pr... only after merge in the separate commit

Reply:
I published this version, that's why I changed it. We talked about it. 
I will now use the local package again, so it will be changed in email and user service.